images/tf-api/Dockerfile

```diff
@@ -11,32 +11,36 @@ RUN apt-get update -qq && apt-get install -y -q \
         software-properties-common \
         unzip \
         zlib1g-dev \
-        python3.6-dev \
-        python3.6-distutils \
         git \
         libsm6 \
         libxext6 \
         libxrender-dev \
         libsndfile1 \
         locales \
-    && apt-get clean -qq && rm -rf /var/lib/apt/lists/* && \
-    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
-    python3.6 get-pip.py && \
-    pip install --upgrade pip && \
-    rm -rf /root/.cache/pip* && \
-    ln -s /usr/bin/python3.6 /usr/local/bin/python
+    && apt-get clean -qq && rm -rf /var/lib/apt/lists/*
 
 RUN locale-gen en_US.UTF-8
 ENV LANG=en_US.UTF-8 LANGUAGE=en_US.en LC_ALL=en_US.UTF-8
 
-ENV PYTHONPATH "${PYTHONPATH}:/src:/mnt/project"
+ENV PYTHONPATH="${PYTHONPATH}:/src:/mnt/project"
+ENV PATH=/opt/conda/bin:$PATH
 
 COPY pkg/workloads/cortex/lib/requirements.txt /src/cortex/lib/requirements.txt
 COPY pkg/workloads/cortex/serve/tf.requirements.txt /src/cortex/serve/requirements.txt
-RUN pip install --upgrade pip && \
-    pip install --no-cache-dir -r /src/cortex/lib/requirements.txt && \
-    pip install --no-cache-dir -r /src/cortex/serve/requirements.txt && \
-    rm -rf /root/.cache/pip*
+COPY pkg/workloads/cortex/environment.yaml /src/cortex/environment.yaml
+
+RUN curl https://repo.anaconda.com/miniconda/Miniconda3-4.7.12.1-Linux-x86_64.sh --output ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
+    /opt/conda/bin/conda install --force-reinstall -c conda-forge python=3.6.9 && \
```

Comment:
Leave a comment on why you did the reinstall

Reply:
Added the comment.